crate/pages/install/cloud/aws.html

```diff
@@ -6,103 +6,69 @@
-
-      <h2>Set up the user data script</h2>
-      <p>Before we can set up the instances we need to supply a configuration script.  This script is run on the instance after it is provisioned.  You don’t have to supply the script if, for example, you some other configuration tool like Puppet, Chef, Salt, etc.  But for this example, we need to put the following code in a file named <code>docker-user-data.sh</code>.</p>
-      <pre><code>#!/bin/bash
-# Update the instances:
-yum -y update
-# Crate the filesystems for Crate's data storage
-mkfs.ext4 /dev/sdb
-mkfs.ext4 /dev/sdc
-# Make the directories to mount the drives
-mkdir /data
-mkdir /data/data{1,2}
-# Mount the drives to the data directories
-mount /dev/sdb /data/data1
-mount /dev/sdc /data/data2
-# Add a crate user and give it ownership of the data directories
-adduser crate -s /sbin/nologin
-chown -R crate.crate /data
-# Install and start Docker
-yum install -y docker
-service docker start
-# Get the latest Crate container for Docker
-docker pull crate:latest
-      </code></pre>
-      <p>Since we aren’t able to use multicast discovery, that’s all we can do right now to set up Crate in advance.  Once we have the instances running, we can then tell Crate where to find the rest of the cluster’s nodes.</p>
-
-      <h2>Set up the instances</h2>
-      <p>Let’s set up three instances.  Don’t forget to put your key name in to the –key-name flag.  Notice that we pass the name of the user data script (above) to the argument –user-data:</p>
-      <pre><code>$ aws ec2 run-instances --count 3 --image-id ami-4b6f650e \
-  --instance-type m3.xlarge \
-  --key-name YOUR_KEY_NAME \
-  --user-data file://docker-user-data.sh \
-  --block-device-mappings "[{\"DeviceName\": \"/dev/xvda\",\"Ebs\":{\"SnapshotId\": \"snap-14cff4d5\", \"VolumeSize\": 8, \"DeleteOnTermination\": true, \"VolumeType\": \"standard\"}}, {\"DeviceName\": \"/dev/sdb\",\"Ebs\":{\"VolumeSize\": 40, \"DeleteOnTermination\": true, \"VolumeType\": \"gp2\"}}, {\"DeviceName\": \"/dev/sdc\",\"Ebs\":{\"VolumeSize\": 40, \"DeleteOnTermination\": true, \"VolumeType\": \"gp2\"}}]"
+<div class="row">
+  <div class="col-sm-8">
+
+    <p>
+      Amazon Web Services (AWS) is the worlds most popular cloud hosting provider with 1000s of options for platforms, applications and infrastructure.
+    </p>
+    <p>
+      An AMI (Amazon Machine Image) is a pre-configured virtual appliance optimized for a particular use. With the Crate AMI, this includes dependencies, optimization and configuration to get Crate running as quickly and efficiently as possible. Using the AMI is our recommended method for running Crate clusters on AWS.
+    </p>
+    <h2>With the AWS Command Line Interface</h2>
+    <p>
+      You can find the Crate AMI via the command line interface:
+    </p>
+    <pre><code>
+aws ec2 describe-images --filters "Name=name,Values=Crate-AWS-AMI-*"
```

Comment:
new naming is: `crate-<VERSION>-<REV>-<BASE_AMI>`
e.g.
`aws ec2 describe-images --filters "Name=name,Values=crate-0.51.1-1-amzn-*"`